packages/synapse-react-client/src/components/error/ErrorPage.tsx

```diff
@@ -4,14 +4,27 @@ import { ReactComponent as UnavailableSvg } from '@/assets/icons/error_page/unav
 import { useSynapseContext } from '@/utils'
 import { Box, Link, Typography } from '@mui/material'
 import { useCallback, useMemo, useState } from 'react'
-import EntityDOIInfo from './EntityDOIInfo'
+import SynapseObjectDOIInfo from './SynapseObjectDOIInfo'
 import SendMessageToEntityOwnerDialog from './SendMessageToEntityOwnerDialog'
+import { DoiObjectType } from '@sage-bionetworks/synapse-client/generated/models/DoiObjectType'
 
 export type ErrorPageProps = {
   type: SynapseErrorType
   message?: string // custom message to report
+  /** @deprecated use `id` and objectType: DoiObjectType.ENTITY */
   entityId?: string
+  /** @deprecated use `version` and objectType: DoiObjectType.ENTITY */
   entityVersion?: number
```

Comment:
Keep these props to avoid a breaking change that affects SynapseWebClient

Reply:
Is there a ticket to update SWC to use id, objectType and version?

Reply:
I'll create one

Reply:
Tracked in SWC-7400